README.md

```diff
@@ -59,6 +57,8 @@ This could be Selenium standalone server, but for local development, you can sen
 
 #### a) Chromedriver
 
+📙 Below you find only simple example, make sure to read our wiki for [more information on Chrome/Chromedriver](https://github.com/php-webdriver/php-webdriver/wiki/Chrome).
```

Comment:
Minor grammatical suggestion:

Below you _will find a_ simple example. Make sure to read our wiki for more information.